src/Product/VersionChecker.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Reflection;
+using System.Text.Json.Serialization;
+
+namespace Azure.DataApiBuilder.Product;
+
+public static class VersionChecker

```

Comment:
nit: is it possible to add some unit tests for the methods here? :)

Reply:
Let's see if anyone likes it first.